.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "4.2.0"
+branches:
+  only:
+    - master
+script:
+  - npm run lint
```

Comment:
Eventually we probably want to change this to run `npm run test`, and run lint as part of the pretest script, but that can wait until we actually _have_ some tests…  😉